{"_id":"91d99160-f7e0-33f1-b51a-873f1fff3ecf","authorName":"Zarat","authorEmailEncrypted":"X9PfVRGLqTS9y4HgM9Knr20TsKn6aGvkWsBDibiTUephmHBOP2NZig+xGasEKCBKAAYHnb5svBF6AXio7gEmrUiApQXbYvu3hDV/6aWNJu5uvT6HOScgJ07uppJEovsmcA8ZiCspFutpemR0z7UniIrgmD6flfZxSlgGu3ZoSae6AJrQ6hv3Tb/qVNU1rsGqi9+j/Dn8gaC9/egdovgnYBk9e3OIKQw77jlCMu7K4bFSLKThaA4cXsVNUkY8xGdF78zhvYM5f4Czxk69ydtAplRUrtk+EYOF6qBH+TUCv5p5qTrx1u9IA+VN6SKT2VHn9x8Rm8dXs9AbKk2ILveTztv2fOXb3IJ11iYNyTG4pycIcG+mIR57itKArAZecXGcDtqBbWmLa8QFMKIIkfeuaB6Y56J1jIrXe79N/zPF8ig16CILB80OqGAdiMBwHXCWmi56ckAdbGf0CX0AdUyns1USQSxPEuxCi2kq+1ApAh4xeLLMln+x5oawZ2opnT08uZ72uoVdYzge/ipKS+YrFE8v1X2on521rABcmcyaymsJdxRgPOOUz835hF2TtDwiZsx4+jL3nenkwfELYYAPiCWRLCgJ3zkjvHd7X6aT9cmSn/h4CI+sX4Le4N7QTfZF4vl7gpp7l25C6J98MfUSmUofcl+Il8uRvZx6XHLw0wc=","authorEmailMD5":"d307e42a92fbce947bfa9fae4400e95f","message":"If you're on 4.5 you can use System.Runtime.ExceptionServices.ExceptionDispatchInfo to store the exception and rethrow it without overwriting the stack trace.","postId":"2014_06_await-in-catch-and-finally-196df97f-fc37-3a25-b9e6-1b95025b3836","replyTo":"e0dc4f0e-ae83-39b8-9ce0-b23ee457470f","date":"2014-06-16T08:23:07Z","timestamp":1402906987,"authorUserId":"disqus:disqus_bCQ4ScqAv7"}